rts/Rendering/Units/UnitDrawer.cpp

```diff
@@ -489,19 +506,44 @@ void CUnitDrawerLegacy::DrawUnitIconsScreen() const
 	sh.Enable();
 	sh.SetUniform("alphaCtrl", 0.05f, 1.0f, 0.0f, 0.0f); // GL_GREATER > 0.05
 
+
+	const float3 camDir = camera->GetDir();
+	// We are going to clamp camDir.y at -0.5, then re-normalize it. 
```

Comment:
why `-0.5` specifically?
```suggestion
	// We are going to clamp camDir.y at -0.5 for reason blablabla, then re-normalize it. 
```

Reply:
consider something along the lines of
```cpp
constexpr float ANGLE_TOO_FLAT = std::sin(-30 * math::DEG_TO_RAD);
```